# This is a BitKeeper generated patch for the following project:
# Project Name: Linux kernel tree
# This patch format is intended for GNU patch command version 2.5 or higher.
# This patch includes the following deltas:
#	           ChangeSet	1.1346  -> 1.1347
#	drivers/ide/pci/siimage.c	1.16    -> 1.17
#
# The following is the BitKeeper ChangeSet Log
# --------------------------------------------
# 03/10/16	noah@caltech.edu	1.1347
# Make several functions in drivers/ide/pci/siimage.c compile regardless
# of CONFIG_PROC_FS.  They are generally applicable, and leaving them out
# prevents building the driver without CONFIG_PROC_FS.
# --------------------------------------------
#
diff -Nru a/drivers/ide/pci/siimage.c b/drivers/ide/pci/siimage.c
--- a/drivers/ide/pci/siimage.c	Thu Oct 16 19:44:07 2003
+++ b/drivers/ide/pci/siimage.c	Thu Oct 16 19:44:07 2003
@@ -35,13 +35,13 @@
 #include "siimage.h"

 #if defined(DISPLAY_SIIMAGE_TIMINGS) && defined(CONFIG_PROC_FS)
-#include <linux/stat.h>
 #include <linux/proc_fs.h>

 static u8 siimage_proc = 0;
 #define SIIMAGE_MAX_DEVS		16
 static struct pci_dev *siimage_devs[SIIMAGE_MAX_DEVS];
 static int n_siimage_devs;
+#endif /* defined(DISPLAY_SIIMAGE_TIMINGS) && defined(CONFIG_PROC_FS) */

 /**
  *	pdev_is_sata		-	check if device is SATA
@@ -120,6 +120,8 @@
 	base |= drive->select.b.unit << drive->select.b.unit;
 	return base;
 }
+
+#if defined(DISPLAY_SIIMAGE_TIMINGS) && defined(CONFIG_PROC_FS)

 /**
  *	print_siimage_get_info	-	print minimal proc information
